### PR TITLE
Adding simple test framework

### PR DESCRIPTION
**Pull request by jhadvig**
@mfojtik @bparees PTAL

### CODE REVIEW

**Comment by mfojtik** [comment]
[test]

**Comment by mnagy** [comment]
FYI: Related to https://github.com/openshift/mysql/pull/2

File: test/run [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+#
+# IMAGE_NAME specifies a name of the candidate image used for testing.
+# The image has to be available before this script is executed.
+#
+IMAGE_NAME=${IMAGE_NAME-openshift/nodejs-010-centos7-candidate}
+
+test_docker_run_usage() {
+  echo "Testing 'docker run' usage..."
+  docker run ${IMAGE_NAME} &>/dev/null
+}
```
Comment (mfojtik) [inline test/run]: docker run --rm

**Comment by bparees** [comment]
lgtm aside from @mfojtik 's comment.

**Comment by jhadvig** [comment]
@bparees @mfojtik updated and rebased.

File: test/run [hunk 2]
```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+#
+# The 'run' performs a simple test that verifies that STI image.
+# The main focus is that the image prints out the base-usage properly.
+#
+# IMAGE_NAME specifies a name of the candidate image used for testing.
+# The image has to be available before this script is executed.
+#
+IMAGE_NAME=${IMAGE_NAME-openshift/nodejs-010-centos7-candidate}
```
Comment (bparees) [inline test/run]: nodejs??

**Pull request merged**